replica/database.hh

```diff
@@ -579,6 +579,8 @@ public:
     // If split is required, then the compaction group of the given tablet is guaranteed to
     // be split once it returns.
     future<> maybe_split_compaction_group_of(locator::tablet_id);
+
+    dht::token_range get_post_split_token_range_of(const dht::token&) const noexcept;
```

Comment:
I think it requires some doc since it's not clear what it is about.

Reply:
Done.